src/tipowof10.c

```diff
@@ -84,6 +88,28 @@ const vui128_t vtipowof10[] = { (vui128_t) (__int128 ) 1ll, /* 10**0 */
   (vui128_t) (__int128 ) 10000000000000000ll, /* 10**16 */
   (vui128_t) (__int128 ) 100000000000000000ll, /* 10**17 */
   (vui128_t) (__int128 ) 1000000000000000000ll, /* 10**18 */
+#ifdef __clang__
```

Comment:
why? add a comment.

Reply:
// Clang does support constant folding for __int128
// Simply not posible to defind a const larger than __UINT64_MAX__
// For vector initialized can use two long long elements.